js/script.js

```diff
@@ -0,0 +1,178 @@
+//getting data from json file and calling tableOeration init method to initialise JS object.
+getData=()=>{
+  let xhr = new XMLHttpRequest();
+  let data;
+  xhr.open('GET','../data/users.json');
+  xhr.overrideMimeType("application/json");
+  //calling init method of tableOperation to get data after loading
+    xhr.onload = function(){
+        data = this.responseText;
+        data = JSON.parse(data);
+        tableOperation.init(data);
+    }
+xhr.send();
+}
+
+//creating IIFE(Immediately Invoked Function Expression) with closure
+tableOperation = (function () {
```

Comment:
1.code formatting and indentation not correct. 
2. Try to follow max character limit for single line.